pvlib/tests/test_pvarray.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+from numpy.testing import assert_allclose
+
+from pvlib import pvarray
+
+
+def test_pvefficiency_adr():
+    g = [1000, 200, 1000, 200, 1000, 200, 0, np.nan]
```

Comment:
Mild preference to make at least one non-nan value a float. Maybe I shouldn't be concerned about it.

Reply:
Sure.